Fix name of a struct field in comment

JulienRouse opened the pull request:
Fix the name in the documentation of the field `isnode` in the struct `large_free_block`.

No change entry needed

JulienRouse committed:
Fix name of a struct field in comment

Fix the name of the field `isnode` in the struct `large_free_block` in the comment documenting it (was `node` before).

JulienRouse commented:
Hi, was just checking [runtime/freelist.c](https://github.com/ocaml/ocaml/blob/trunk/runtime/freelist.c#L868) and noticed the mismatch between the name of the field in the documentation and in the struct. 

If it is intentional or forgot step in the Contributing.md, please let me know!

dra27 commented on runtime/freelist.c:
```diff
@@ -879,7 +879,7 @@ static int bf_small_map = 0;
-     - node flag
+     - isnode flag
      - left son
      - right son
```
While here, out of curiosity of @damiendoligez, why is "son" used? FWIW is the more neutral "child" still appropriate?

damiendoligez replied:
I guess it's a cultural thing. In French, we have no word for "sibling" so for trees instead of child/sibling we still use *fils*/*frère*. Then I naturally translate into english as son/brother. If you want to change these comments I won't object.

dra27 replied:
Ah, I thought that might be it, thanks!

JulienRouse replied:
Should I rename "son" to "child" then or is it ok like this?

dra27 replied:
I think it’s slightly clearer as `child`, so yes, please

JulienRouse committed:
Rename "son" to "child" in struct documentation

Change the name to "child" in the documentation of the struct "large_free_block" to use a more common english term.

JulienRouse committed:
Fix deleted newline

No change entry needed

xavierleroy reviewed (approved):
Looks good to me.  No reason to drag it out.
